projects/sandbox/src/app/_models/navigation-orbs.ts

```diff
@@ -1,4 +1,5 @@
 export enum DisplayedTier {
+  NONE = -1,
   CONTENT = 0,
   METADATA
```

Comment:
My suggestion is that `METADATA` should also have an explicit value defined for readability.